src/server/rest/v1/service/TenantService.ts

```diff
@@ -187,6 +188,25 @@ export default class TenantService {
     }
   }
 
+  public static async createInitialUserSetting(tenantID: string): Promise<void> {
+    // Check for settings in db
+    const userSettings = await SettingStorage.getUserSettings(tenantID);
+    if (Utils.isEmptyObject(userSettings)) {
+      // Create new user setting with account activation param
+      const settingsToSave = {
+        identifier: UserSettingsType.USER,
+        content: {
+          type: UserSettingsContentType.USER,
+          user: {
+            manualAccountActivation: false
```

Comment:
@ClaudeROSSI here is the new tenant user setting

I added "false" otherwise I will need to add some other logic to handle the fact the first user (the one that created the tenant) will be blocked as its accoount will be in "manual activation mode" == inactive == he won't be able to even log to the app.